bonan_richards_eqn/get_bonan_data.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# This script downloads a specific version of a Matlab script from a GitHub repository
```

Comment:
Maybe we should note the date on which we accessed the script and ran it? Just in case he updates his code

Reply:
nevermind ! you checkout a specific commit.